### PR TITLE
New fast api

### PR DESCRIPTION
**Pull request by milsman2**
New FASTApi skeleton with configurations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,34 +20,34 @@ TBD
...
 
 ## 🤝 Contributing
 
 For information on contributing to this project, please see the [contributing guide](CONTRIBUTING.md) and our [code of conduct](CODE_OF_CONDUCT.md).
 
 ## 🔗 Related
 
-* [Open Brewery DB Website & Documentation](https://github.com/chrisjm/openbrewerydb-gatsby)
-* [Open Brewery DB Dataset](https://github.com/openbrewerydb/openbrewerydb)
+- [Open Brewery DB Website & Documentation](https://github.com/chrisjm/openbrewerydb-gatsby)
```
Comment (chrisjm) [inline README.md]: This should be updated to `[Open Brewery DB Website & Documentation](https://github.com/openbrewerydb/openbrewerydb-sveltekit)`

File: app/core/config.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+logger = logging.getLogger(__name__)
+
+
+class EnvironmentEnum(str, Enum):
+    """
+    Enumerate the application environments
+    """
+
+    PRODUCTION = "production"
+    LOCAL = "local"
```
Comment (chrisjm) [inline app/core/config.py]: Can this be `DEVELOPMENT = "development"` for consistency or is that not Pythonic? (disclaimer: I haven't kept up with Python for over a year, so I'm not sure what's standard these days 😄 )

File: app/core/config.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+    TITLE: str = "Open Brewery DB REST API"
+    DESCRIPTION: str = "A REST API for Open Brewery DB"
+    JWT_SECRET: str = secrets.token_urlsafe(32)
+    ACCESS_TOKEN_EXPIRE_MINUTES: int = 60 * 24 * 8
+    ALGORITHM = "HS256"
+    ENVIRONMENT: EnvironmentEnum
+    DEBUG: bool = False
+    TESTING: bool = False
+    TIMEZONE: str = "CDT"
```
Comment (chrisjm) [inline app/core/config.py]: Just a suggestion, fine to leave it as is for now, but eventually let's make this UTC and/or add a note in the `README` so people know they need to change it.

File: app/core/config.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    # DB_USER: str = os.environ["DB_USER"]
+    # DB_PWD: str = os.environ["DB_PWD"]
+    # DB_HOST: str = os.environ["DB_HOST"]
+    # DB_NAME: str = os.environ["DB_NAME"]
+    # DATABASE_URL: Optional[
+    #     PostgresDsn
+    # ] = f"postgresql://{DB_USER}:{DB_PWD}@{DB_HOST}:5432/{DB_NAME}"
+    # DB_ECHO_LOG: bool = False
+    # Api V1 prefix
+    API_V1_STR = "/api/v1"
```
Comment (chrisjm) [inline app/core/config.py]: Can we shorten this to `/v1`? `/api/` feels a little redundant.

File: app/api/api_v1/api.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+Import all routers together
+"""
+from fastapi import APIRouter
+from app.api.api_v1.routes import healthcheck
+
+api_router = APIRouter()
+
+
+api_router.include_router(
+    healthcheck.router, prefix="/healthcheck", tags=["healthcheck"]
```
Comment (chrisjm) [inline app/api/api_v1/api.py:11]: Move this to top-level if we're good with removing the `/api/` prefix?

**Review by chrisjm — COMMENTED** [review]
Thank you so much @milsman2! I haven't tried to run it yet, but it looks fine to my non-Python-dev 😆 . I have a couple of comments, questions, and requests. Please feel free to push back.

File: app/core/config.py (hunk 4 above)
Comment (milsman2) [reply app/core/config.py]: If you don't mind, this is one I would like to keep. That seems to be one of the more popular methods for versioning the REST API itself.

File: app/api/api_v1/api.py (hunk 5 above)
Comment (milsman2) [reply app/api/api_v1/api.py:11]: See comment below. If it's all right I would like to leave the structure as is based on most of the examples I have read. If that is not okay, it can definitely be changed.

File: app/core/config.py (hunk 4 above)
Comment (chrisjm) [reply app/core/config.py]: I get it, but I'm envisioning:

`https://api.openbrewerydb.org/v1/breweries`

As opposed to:

`https://api.openbrewerydb.org/api/v1/breweries`

If it's _just_ the popular way, I'd prefer the OBDB example to be an alternative to show devs different ways to do things to get the same effect, especially when considering production environments, separation of systems, and subdomains.

Comment (chrisjm) [reply app/core/config.py]: Thank you @milsman2 !

**Review by chrisjm — APPROVED** [review]
Thanks for the updates @milsman2 ! I wasn't able to get this loaded, so eventually I'll want to get some Getting Started instructions in the README on how to do that for those rusty in their Python usage. 😅  All good for now, Thanks again! 🍻
